lib/walder.js

```diff
@@ -231,6 +232,14 @@ module.exports = class Walder {
       }
     }
 
+    // Validate default error pages
+    this.logger.verbose('Validating the default error pages');
+    if (validator.validate({routeInfo: new RouteInfo('default error pages', 'any'), htmlInfoDictionary: defaultErrorPages})) {
+      Object.keys(validator.prevErrors).forEach((type) => {
```

Comment:
It's better to work with a getter instead of the variable directly for `validator.prevErrors`.

Reply:
That was copied code of a block occuring in the loop above the mentioned lines.
But that whole block of code is bad practice.
I fetches a property outside the defined interface of Validator, just for logging purposes.
Now that the validators have non-static member functions, they can directly access a logger, given to them at construction time, and do the logging themselves right in the validate() method.
I'll change the code to do that.